### PR TITLE
Documentation for installing .NET Core & Desktop

### PR DESCRIPTION
**Pull request by thesmithee**
Added processes for installing .NET Core and .NET Desktop framework on
all supported OS-es.

/cc @richlander

### CODE REVIEW

File: docs/getting-started/installing-core-macosx.rst [hunk 1]
```diff
@@ -1,7 +1,153 @@
 .. include:: /stub-topic.txt
 
-|stub-icon| Installing .NET Core on MacOSX 
-==========================================
+Installing .NET Core on MacOSX
```
Comment (richlander) [inline docs/getting-started/installing-core-macosx.rst]: It's "OS X"

http://en.wikipedia.org/wiki/OS_X

File: docs/getting-started/installing-desktop.rst [hunk 2]
```diff
@@ -1,7 +1,25 @@
 .. include:: /stub-topic.txt
 
-|stub-icon| Installing .NET Framework
-=====================================
+Installing .NET Desktop Framework
```
Comment (richlander) [inline docs/getting-started/installing-desktop.rst]: It's call ".NET Framework". "Desktop" is a colloquialism.

Do we need this documentation in the .NET Core docs?

**Comment by richlander** [comment]
LGTM (modulo comments)

File: docs/getting-started/installing-desktop.rst (hunk 2 above)
Comment (blackdwarf) [inline docs/getting-started/installing-desktop.rst]: Depends. Seemed like a good idea at the time, but now not so much. :)

**Pull request merged**